src/main/docker/README.md

```diff
@@ -26,8 +26,8 @@ Build Eiffel Gerrit Plugin Docker image:
 
 ```bash
 cd (git root dir)
-export URL=target/$(ls target/ | grep '^eiffel-gerrit-plugin-[0-9]*.[0-9]*.[0-9]*.jar')
-docker build -t eiffel-gerrit-plugin --build-arg URL=./${URL} -f src/main/docker/Dockerfile .
+export PLUGIN_LOCATION=target/$(ls target/ | grep '^eiffel-gerrit-plugin-[0-9]*.[0-9]*.[0-9]*.jar')
```

Comment:
I think one of the reasons we used URL before was so we could keep the build process generic for several Eiffel component Docker files, and we could then re-use the same automated script for building Docker images etc. 

This PLUGIN_LOCATION is very specific to this particular component, but I'm hesitant since I'm not sure if we need to keep it generic for all Eiffel components ... How about using JAR_LOCATION?

Reply:
I think the cons outweigh the use. I have changed the argument in 2e9b0dc